### PR TITLE
git-town: 11.1.0 -> 13.0.0

### PR DESCRIPTION
**Pull request by gabyx**
## Description of changes

https://github.com/git-town/git-town/releases/tag/v13.0.0

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by gabyx** [comment]
@allonsy, @blaggaca: FYI, to review.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6879,6 +6879,15 @@
     githubId = 1313787;
     name = "Gabriella Gonzalez";
   };
+  gabyx = {
+    email = "gnuetzi@gmail.com";
+    github = "gabyx";
+    githubId = "647437";
```
Comment (NickCao) [inline maintainers/maintainer-list.nix:6885]:
```suggestion
    githubId = 647437;
```

Should be an int.

**Comment by gabyx** [comment]
@NickCao : Added some release info due to breaking changes.

File: nixos/doc/manual/release-notes/rl-2405.section.md [hunk 2]
```diff
@@ -145,6 +145,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `unrar` was updated to v7. See [changelog](https://www.rarlab.com/unrar7notes.htm) for more information.
 
+- `git-town` was updated from `v11` to `v13.`. See the [changelog](https://github.com/git-town/git-town/blob/main/CHANGELOG.md#1300-2024-03-22) for breaking changes.
```
Comment (SuperSandro2000) [inline nixos/doc/manual/release-notes/rl-2405.section.md:148]:
```suggestion
- `git-town` was updated from `11` to `13.`. See the [changelog](https://github.com/git-town/git-town/blob/main/CHANGELOG.md#1300-2024-03-22) for breaking changes.
```

File: pkgs/applications/version-management/git-town/default.nix [hunk 3]
```diff
@@ -63,7 +64,7 @@ buildGoModule rec {
     description = "Generic, high-level git support for git-flow workflows";
     homepage = "https://www.git-town.com/";
     license = licenses.mit;
-    maintainers = with maintainers; [ allonsy blaggacao ];
+    maintainers = with maintainers; [ allonsy blaggacao gabyx];
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-town/default.nix:67]:
```suggestion
    maintainers = with maintainers; [ allonsy blaggacao gabyx ];
```

Comment (gabyx) [reply pkgs/applications/version-management/git-town/default.nix:67]: Isn't there a format rule or make targer to do this?

**Comment by gabyx** [comment]
@SuperSandro2000 : Do you know somebody to talk to about the formatting/linting thing in the nixpkgs repo? I think this repo is central/important enough to really have something in place.

File: pkgs/applications/version-management/git-town/default.nix (hunk 3 above)
Comment (SuperSandro2000) [reply pkgs/applications/version-management/git-town/default.nix]: Formatting should be foremost locally consistent. If a list starts with a space, it should also end with one.

alejandra mostly produces weird code and isn't really recommendable. 

There is a formatting RFC that is being worked on.

**Comment by SuperSandro2000** [comment]
Please rebase after #298194 got merged.

**Pull request merged**